auth/user.go

```diff
@@ -233,7 +248,7 @@ func (user *userImpl) RevokedChannels(since uint64) RevokedChannels {
 		for chanName, history := range princ.ChannelHistory() {
 			if !accessibleChannels.Contains(chanName) {
 				for _, entry := range history.Entries {
-					if entry.StartSeq <= since && entry.EndSeq > since {
+					if entry.StartSeq <= checkSeq && entry.EndSeq > checkSeq || (entry.StartSeq <= triggeredBy && entry.EndSeq > triggeredBy) {
```

Comment:
Is it functionally equivalent to set checkSeq == triggeredBy above when triggeredBy !=0, and change this check to:
`if entry.StartSeq <= checkSeq && (entry.EndSeq > checkSeq || entry.EndSeq == triggeredBy)`?  I feel that simplifies and better represents the logic, which is "if the current changes lastSeq was in a window where the user had channel access, or we're in an interrupted revocation backfill for the end of one of those windows"